src/implementations/transaction/transaction.test.ts

```diff
@@ -0,0 +1,69 @@
+import { cancelTransaction } from "./transaction";
+import { getWallet } from "../utils/wallet";
+import path from "path";
+import signale from "signale";
+import { BigNumber } from "ethers";
+jest.mock("../utils/wallet");
+
+const mockGetWallet = getWallet as jest.Mock;
+
+describe("document-store", () => {
+  describe("cancelTransaction", () => {
+    const signaleSuccessSpy = jest.spyOn(signale, "success");
+    const signaleErrorSpy = jest.spyOn(signale, "error");
+
+    it("success in canceling transaction using nonce and gas", async () => {
```

Comment:
If we mochi the transaction then we test nothing. Why do we mock here ? Send transaction and get wallet